src/sema/ast.rs

```diff
@@ -422,6 +422,18 @@ impl Function {
         matches!(self.mutability, Mutability::Payable(_))
     }
 
+    /// Does this function have an @payer annotation
```

Comment:
Should we add a question mark to the end of questions?

Reply:
It's a comment, it does not matter

Reply:
This is clearly a nitpick, but the sentence is awkward without the question mark. At Google, I was required to use the correct punctuation in all my comments. This is why I commented here. What do you think?

Reply:
It's a nitpick and it's immaterial.

If the clarity of the comment was affected, then it would matter.  `Does this function have an @payer annotation` is perfectly clear. It simply does not matter much.

However, having said all that, I don't mind if you point out problems with punctuation or other nitpicks, as long as they are not marked "changes required". The improvement is so minor that it is not really worth much of our time.

This is generally how open source projects do reviews, it is not specific to solang.